recipes/letsencrypt.rb

```diff
@@ -14,6 +14,11 @@
 EOF
 end
 
+remote_file '/tmp/DigiCertGlobalRootG2.crt' do
```

Comment:
If you check a few lines above you'll see that this recipe is only for `hops.site` It's not included anywhere else. This is not the correct place.